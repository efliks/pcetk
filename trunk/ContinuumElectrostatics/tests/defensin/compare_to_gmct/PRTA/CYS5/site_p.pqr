ATOM      0  CB  CYS     5      20.705  32.435   6.605  -0.11000   1.700      0      0      0      0
ATOM      0  HB1 CYS     5      20.282  33.416   6.290   0.09000   1.000      0      0      0      0
ATOM      0  HB2 CYS     5      20.289  31.665   5.918   0.09000   1.000      0      0      0      0
ATOM      0  SG  CYS     5      20.230  32.044   8.311  -0.23000   1.800      0      0      0      0
ATOM      0  HG1 CYS     5      18.921  32.128   8.128   0.16000   0.300      0      0      0      0
