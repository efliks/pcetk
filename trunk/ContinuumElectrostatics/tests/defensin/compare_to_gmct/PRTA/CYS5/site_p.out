Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named scratch/PRTA/CYS5/site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (24.1776, 30.1496, 6.10909)) 121 2
Finer level:    (20.0853, 32.3377, 7.05061) 101 1
Finer level:    (20.0853, 32.3377, 7.05061) 101 0.5
Finest level:   (20.0853, 32.3377, 7.05061) 101 0.25


Self energy of scratch/PRTA/CYS5/site_p in three dielectric environment = 23.7333
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/CYS5/site_p with background charge set in three dielectric environment = -0.268583
(probably in kcal/mole)



Interaction energies of scratch/PRTA/CYS5/site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.000815164
0   1 : 0.00466821
1   0 : -0.00617505
1   1 : 0.0565625
2   0 : 0.00110278
2   1 : 0.00782163
3   0 : 47.4583
3   1 : 88.0305
4   0 : -0.0170163
4   1 : -0.00173401
5   0 : 0.00330915
5   1 : 0.0579433
6   0 : 0.00435517
6   1 : 0.040404
7   0 : -0.00299403
7   1 : -0.00012046
8   0 : -0.0230221
8   1 : -0.00167582
9   0 : 0.000219722
9   1 : 0.00758124
10   0 : 0.522685
10   1 : 2.38387
11   0 : -0.00635266
11   1 : -0.0412917
12   0 : 0.00299601
12   1 : -0.000124501
13   0 : 0.0104168
13   1 : 0.158953
14   0 : 0.00902139
14   1 : 0.0448651
Total runtime: 10.36 s user 0.05 s system 
